programs/CMakeLists.txt

```diff
@@ -13,12 +13,6 @@ option (ENABLE_CLICKHOUSE_SERVER "Server mode (main mode)" ${ENABLE_CLICKHOUSE_A
 option (ENABLE_CLICKHOUSE_CLIENT "Client mode (interactive tui/shell that connects to the server)"
     ${ENABLE_CLICKHOUSE_ALL})
 
-# Don't create self-extracting clickhouse for split build
```

Comment:
The option `SPLIT_SHARED_LIBRARIES` should be removed from the top-level CMakeLists and src/CMakeLists.txt.